internal/commands/main.go

```diff
@@ -90,11 +90,18 @@ type FetchResultError struct {
 }
 
 func (c Commands) fetchAllFeeds(noCacheOverride bool) ([]rss.RSS, error) {
+
 	var (
 		rsss []rss.RSS
 		wg   sync.WaitGroup
 	)
 
+	if c.config.Preview != "" {
```

Comment:
If we change this to just set config.feeds based on the preview url, then we can use the existing fetching/caching logic.

```go
   feeds := c.config.Feeds

   // if we have a preview url, ignore the config URLs
   if c.config.Preview != nil {
     feeds := []Feed{URL: c.config.Preview}
   }
```